Solutions/Marain.UserNotifications.Specs/Marain.UserNotifications.Specs.csproj

```diff
@@ -55,6 +56,10 @@
       <Visible>$(UsingMicrosoftNETSdk)</Visible>
       <CodeBehindFile>%(RelativeDir)%(Filename).feature$(DefaultLanguageSourceExtension)</CodeBehindFile>
     </SpecFlowFeatureFiles>
+    <SpecFlowFeatureFiles Update="Features\ManagementApiClient\BatchNotificationDeliveryStatusUpdate.feature">
```

Comment:
I thought with the current SpecFlow tooling we weren't supposed to get these warts in the `.csproj`?

Reply:
I think these might turn up as a result of copy-pasting feature files. I've cleaned them up.